README.md

```diff
@@ -0,0 +1,24 @@
+# Palette Picker
+The app allows you to pick color palettes and save them to a relational database.
+
```

Comment:
Can we also add data structure of project object and palette object to documentation?